scrapy_webarchive/wacz.py

```diff
@@ -77,7 +84,35 @@ def cleanup_files(self, *files: str) -> None:
     def get_wacz_fname(self) -> str:
         """Generate WACZ filename based on the WARC filename."""
 
-        return f"{self.collection_name}-{get_current_timestamp()}.wacz"
+        return f"{self.collection_name}-{get_formatted_dt_string(format=TIMESTAMP_DT_FORMAT)}.wacz"
+    
+    def write_datapackage(self, zip_file: zipfile.ZipFile):
+        package_dict = {
+            "profile": "data-package",
+            "title": self.collection_name,
+            "description": "This is the web archive generated by a Scrapy extension for the " \
+                f"{self.collection_name} spider. It is mainly for scraping purposes as it does not contain " \
+                "any js/css data. Though it can be replayed as bare HTML if the site does not depend on JS.",
+            "created": get_formatted_dt_string(format=WARC_DT_FORMAT),
+            "software": f"scrapy-webarchive v{scrapy_webarchive_version}, using Scrapy v{scrapy_version}",
```

Comment:
Let's check what is elsewhere used as software string here; user-agent strings usually have something like `<name>/version`.

Reply:
Comparing a few examples I think the `<name>/<version>` notation is the cleanest. Other considerations were:

`scrapy-webarchive (<version>), Scrapy (<version>)`
`scrapy-webarchive;<version>, Scrapy;<version>`
`scrapy-webarchive v(<version>), Scrapy v(<version>)` (current)